components/git/land.js

```diff
@@ -152,8 +157,14 @@ async function main(state, argv, cli, req, dir) {
       cli.log('run `git node land --abort` before starting a new session');
       return;
     }
-    session = new LandingSession(cli, req, dir, argv.prid);
+    session = new LandingSession(cli, req, dir, argv.prid, argv.backport);
     const metadata = await getMetadata(session.argv, cli);
+    if (argv.backport) {
+      const split = metadata.metadata.split('\n')[0];
+      if (split === 'PR-URL: ') {
+        cli.error('Commit message is missing PR-URL');
```

Comment:
I don't know if this covers all cases, but it seems to me that any PR that has a commit message with `PR-URL: ` already in it is a backport by definition, so doesn't need the argv switch. Or do we have other cases?

Reply:
there are other cases